data_conditioner.py

```diff
@@ -36,7 +38,7 @@
 r0 = 2.81794E-15    # [m]      classical electron radius
 h = 4.135667E-15    # [eV s]    plancks constant
 c = 2.997945E8      # [m s^-1] Speed of light
-NA = 6.02240E23     # []       Avagadros number 
+NA = 6.02240E23     # []       Avagadros number
 
 data_dicts = [Si_data, C_data]
```

Comment:
Down the road it would be nice to have these data dicts come from a config file that can easily be populated by science staff.  Not really a crucial issue at the moment since AT2L0 will only have silicon and diamond filters.   When they eventually scrutinize their transmission measurements these will be the first parameters they should check.

Reply:
Good to note :+1: